src/Sylius/Behat/Context/Api/Admin/ManagingShipmentsContext.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Component\Core\Formatter\StringInflector;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Customer\Model\CustomerInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingShipmentsContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    public function __construct(ApiClientInterface $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * @When I browse shipments
+     */
+    public function iBrowseShipments(): void
+    {
+        $this->client->index('shipments');
+    }
+
+    /**
+     * @Then I should see( only) :count shipment(s) in the list
+     * @Then I should see a single shipment in the list
+     */
+    public function iShouldSeeCountShipmentsInList(int $count = 1): void
+    {
+        Assert::same($this->client->countCollectionItems(), $count);
+    }
+
+    /**
+     * @Then I should see the shipment of order :orderNumber as :shippingState
+     */
+    public function iShouldSeeTheShipmentOfOrderAs(string $orderNumber, string $shippingState): void
+    {
+        Assert::true(
+            $this->isShipmentWithOrderNumberAndState($orderNumber, $shippingState),
+            sprintf('Shipment for order %s with state %s does not exist', $orderNumber, $shippingState)
+        );
+    }
+
+    /**
+     * @Then /^I should see shipment for (the "[^"]+" order) as (\d+)(?:|st|nd|rd|th) in the list$/
+     */
+    public function iShouldSeeShipmentForTheOrderInTheList(string $orderNumber, int $position): void
+    {
+
+        Assert::true(
+            $this->client->hasItemOnPositionWithValue(--$position, 'order', sprintf('/new-api/orders/%s', $orderNumber)),
```

Comment:
Please, refactor to the same approach, as Grzegorz used: https://github.com/Sylius/Sylius/pull/11215/files#diff-eeb3d6c16e3f1878e2378c83d066a24cR145

With his approach, we are not coupled to IRI definition and future refactor will be easier.

You may need to add `sylius.behat.context.transform.order` to suite definition